app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ReviewsController < ApplicationController
+  before_action :set_book
+  before_action :set_review, only: [ :edit, :update, :destroy ]
+
+  def create
+    @review = @book.reviews.create(review_params)
+    if @review.update(review_params)
```

Comment:
Is it Redundant update?
```suggestion
     @review = @book.reviews.new(review_params)
    if @review.sabe
```